scripts/snls/snls.py

```diff
@@ -61,16 +61,14 @@
 
 _LICENSE = "GNU General Public License v3.0"
 
-_VERSION = "0.0.1"
+_VERSION = "1.0.0"
 
 _STR_FEATURES = [
     "object_id",
     "obj_type"
 ]
 
 _FLOAT_FEATURES = [
-    "ra", 
-    "dec", 
```

Comment:
These shouldn't be dropped unless I'm missing something.

Reply:
we are not including ra, dec as features